isis/src/system/apps/kerneldbgen/SpiceDbGen.cpp

```diff
@@ -224,10 +224,9 @@ PvlGroup SpiceDbGen::AddSelection(FileName fileIn, double startOffset, double en
 
       //find the correct coverage window
       if (currFile == "SPK") {
-        //  2000 is the max coverage window size for an SPK kernel
-        SPICEDOUBLE_CELL(cover, 2000);
+        SPICEDOUBLE_CELL(cover, 50000);
         ssize_c(0, &cover);
-        ssize_c(2000, &cover);
+        ssize_c(200000, &cover);
```

Comment:
Should this be the same number as in line 227?

Reply:
Probably.

Reply:
Thanks for catching that. Fixed!